js/thankyou.js

```diff
@@ -36,7 +37,7 @@ var typingEffect = function typingEffect() {
    setTimeout(function () {
      app.innerHTML = '';
      typingEffect();
-   }, 1000);
```

Comment:
@Ryan10145  Why did you change the timeout??

Reply:
@sk9331657 Specifically when I was trying to read the Chinese, the language would change when I was halfway through. So I changed the timeout since I felt that for more complicated character based languages, there wasn't enough time given to read.

Reply:
Ok @Ryan10145